source/TextBlade.Core/UserCode/CodeBehindRunner.cs

```diff
@@ -6,13 +6,13 @@ namespace TextBlade.Core.Game;
 
 public static class CodeBehindRunner
 {
-    private static Dictionary<string, Type> _classNameToType = new();
+    private static readonly Dictionary<string, Type> _classNameToType = new();
 
     static CodeBehindRunner()
     {
         // Cache all class names and types. Maybe a bad idea. How much of the game will the player really see?
-        var assembly = Assembly.GetEntryAssembly();
-        RegisterAssemblyClasses(assembly);
+        if (Assembly.GetEntryAssembly() is {} assembly)

```

Comment:
I don't understand this code. Is it possible `GetEntryAssembly` is null? Is that what you're checking for?

Reply:
public static System.Reflection.Assembly? GetEntryAssembly ();

Yes, it can be null. That code checks if the result is not null and, if so, assigns it to the assembly variable